### ISSUE TITLE
which translation platform do you use?

### ISSUE THREAD

**Issue by Jibec** [storaged-project/udisks#719]
Hello, the Fedora project migrates its translation platform to Weblate [1].

I can see your project is both on GNOME and Fedora translation platform:
https://l10n.gnome.org/module/udisks/
https://fedora.zanata.org/project/view/udisks

As a contributor, having all of your project at the same place makes more sense to me. But this is an upstream decision.
Whatever you decide, could you please lock the Zanata project?

If you want to go with the Fedora translation platform, here are the questions:
This tool directly interact with your git repository, and requires us to know:
* [mandatory] which branch is your development branch?
* [mandatory] have you merged latest translation from Zanata and locked the project?
* [info] Weblate will handle updates when pot file changes, don't edit po files for this [2]
* [optional] what is the license of translation? (please use a code from https://spdx.org/licenses/)
* [optional] do you have any announcement/warning you would like to display to the translators? (it will be displayed in Weblate)
* [optional] do you need us to activate any specific checks? (this is a setting per component [3])
* [optional] do you need us to automatically detect new translation files? (typical usecase: website translation with one translation file per page)

Please note:
* For github hosted projects, Weblate open pull request. For other project, you'll have to add a key to allow commits.
* In Weblate's vocable, one project is a group of component. Each component is a translation file. You can have many projects or many components or both.
* You can change your mind over time, just reach trans@lists.fedoraproject.org

[1] https://communityblog.fedoraproject.org/fedora-localization-platform-migrates-to-weblate/ 
[2] https://docs.weblate.org/en/latest/admin/continuous.html#avoiding-merge-conflicts
[3] https://docs.weblate.org/en/latest/user/checks.html#translation-checks

**Comment by fitojb** [comment]
@storaged-project Please do not leave us translators in limbo. In the mean time, do you accept PRs?

**Comment by tbzatek** [comment]
Umm... yes, too late for the 2.9.1 release. Let me have a look what other options we have...

**Comment by Jibec** [comment]
Zanata will be shut down in a few weeks now.

Storaged project already have these project on Weblate: 
https://github.com/storaged-project/blivet/issues/821
https://github.com/storaged-project/blivet-gui/issues
https://github.com/storaged-project/libbytesize/issues/63

I feel like it is confusing to have multiple translations process for storaged and suggest you to ask to GNOME translation team to stop hosting it.

You may get help from @vojtechtrefny who did it for rest of your projects.

**Comment by Jibec** [comment]
Kind reminder, we would all win to have the all the ecosystem in the same place.
ping @mscherer

**Comment by wsxy162** [comment]
Nowhere to contribute translations for now?

**Comment by tbzatek** [comment]
I'm happy to announce that we've been working with the Gnome Infrastructure team to connect the Gnome Translation Platform with our external Github repo. See the related discussion: https://gitlab.gnome.org/Infrastructure/damned-lies/-/issues/383

The translation module page is here: https://l10n.gnome.org/module/udisks/

Things are still in experimental phase, however we've merged some new strings earlier today and I can see them in the POT file on the Gnome side already.

We just need to test that committing translations do work the other way around, kindly asking people watching the place here to translate something :-)

Once proved working, I'll update the TRANSLATING document in the source tree.

**Comment by tbzatek** [comment]
(Cc: our recent translators: @haaninjo, @piotrdrag, @NorwayFun, @yurchor)

**Comment by tbzatek** [comment]
> GNOME l10n page says to create PR. "Daemon, tools and libraries to access and manipulate disks and storage devices. To submit your translation, [create a pull request](https://github.com/storaged-project/udisks/pulls)."

Oh yes, we need to change that as well, along with maintainer contacts. Does the system allow you to submit your translation anyway? I'm not a member of any translation team on that side.

**Comment by NorwayFun** [comment]
> > GNOME l10n page says to create PR. "Daemon, tools and libraries to access and manipulate disks and storage devices. To submit your translation, [create a pull request](https://github.com/storaged-project/udisks/pulls)."
> 
> Oh yes, we need to change that as well, along with maintainer contacts. Does the system allow you to submit your translation anyway? I'm not a member of any translation team on that side.

To submit translation - yes.
https://l10n.gnome.org/vertimus/udisks/master/po/ka/
But no "push to repository" in menu.

**Comment by atriwidada** [comment]
> > > GNOME l10n page says to create PR. "Daemon, tools and libraries to access and manipulate disks and storage devices. To submit your translation, [create a pull request](https://github.com/storaged-project/udisks/pulls)."
> > 
> > 
> > Oh yes, we need to change that as well, along with maintainer contacts. Does the system allow you to submit your translation anyway? I'm not a member of any translation team on that side.
> 
> To submit translation - yes. https://l10n.gnome.org/vertimus/udisks/master/po/ka/ But no "push to repository" in menu.

same here, https://l10n.gnome.org/vertimus/udisks/master/po/id/
seems that configuration to allow push to github repo is not configured properly yet

**Comment by atriwidada** [comment]
> > > GNOME l10n page says to create PR. "Daemon, tools and libraries to access and manipulate disks and storage devices. To submit your translation, [create a pull request](https://github.com/storaged-project/udisks/pulls)."
> > 
> > 
> > Oh yes, we need to change that as well, along with maintainer contacts. Does the system allow you to submit your translation anyway? I'm not a member of any translation team on that side.
> 
> To submit translation - yes. https://l10n.gnome.org/vertimus/udisks/master/po/ka/ But no "push to repository" in menu.

same here, https://l10n.gnome.org/vertimus/udisks/master/po/id/
seems that configuration to allow push to github repo is not configured properly yet

**Comment by tbzatek** [comment]
Thanks, this is the kind of feedback I was looking for. I'll follow up on this issue with Gnome folks.

**Comment by atriwidada** [comment]
tried to execute "submit to repository" via l10n, got this error:

> An error occurred during applying your action: The commit failed. The error was: “[Errno 1] Command: "['git', 'push', 'origin', 'master']", Error: Warning: Permanently added 'github.com' (ED25519) to the list of known hosts. remote: error: GH006: Protected branch update failed for refs/heads/master. remote: error: Changes must be made through a pull request. To github.com:storaged-project/udisks.git ! [remote rejected] master -> master (protected branch hook declined) error: failed to push some refs to 'github.com:storaged-project/udisks.git' ”

**Comment by tbzatek** [comment]
Yeah, this is still pending, moving rather slow: https://gitlab.gnome.org/Infrastructure/damned-lies/-/issues/383

**Comment by rffontenelle** [comment]
Close this as solved? Pushing translations via Damned Lies seems to be implemented now as can be seen by commit history and the issue linked above.

**Comment by tbzatek** [comment]
Translations are working nicely and new strings are being pushed in the repo just fine. I just need to update https://github.com/storaged-project/udisks/blob/master/TRANSLATING
